controls/radtreelistview/radtreeliestview-getting-started.md

```diff
@@ -10,37 +10,19 @@ position: 3
 
 # Getting Started
 
+This article will guide you through the process of creating a sample application with __RadTreeListView__.
 
-
-The __RadTreeListView__ is a control used to display hierarchical data in a tabular format. In order to achieve this, the __RadTreeListView__ combines the functionality of the __RadGridView__ control with the ability to display data in a tree view manner. This is implemented by directly inheriting the __RadGridView__ and implementing the tree view behavior.
-
-The __RadTreeListView__ displays the data in the same way as the __RadGridView__. The difference is that each row plays the role of a tree node, which can be expanded and collapsed.
-
->Please note that the __RadTreeListView__ and the [Hierarchical RadGridView]({%slug gridview-hierachy-overview%}) are two different things. The first displays the data in a tree view manner in one and the same table. The second one displays the hierarchical data in nested __RadGridViews__. 
-
-
->tip Consider using the __RadTreeListView__ for displaying only a homogeneous data. For heterogeneous data use the [Hierarchical RadGridView]({%slug gridview-hierachy-overview%}).
-
-The thing that you should use in order to display your data in the __RadTreeListView__ is the __ChildTableDefinitions__ property. In the context of the __RadTreeListView__ control, you shouldn't use the __GridViewTabelDefinition__ class, but the __TreeListViewTableDefinition__ one. It exposes an __ItemSource__ property which should be bound to the property that represents the next level of hierarchy. 
-
->This property should have the same name at each level of the hierarchy. 
-
->As the hierarchical data in the __RadTreeListView__ is displayed in one and the same table, each level of the hierarchy will share one and the same set of columns. This means that your data should provide the needed members at each level of the hierarchy.
-
->caution Binding to the __IsExpanded__ property of __TreeListViewRow__ is not fully supported. You can consider using [IsExpandedBinding]({%slug radtreelsitview-how-to-control-expand-settings%}) property instead.
-
->tipYou can also check out the __RadGridView__'s [Getting Started section]({%slug gridview-getting-started2%}).
-        
-
-This topic will explain you how to:
+* [Assembly References] (#assembly-references)
 
 * [Add RadTreeListView to your application](#add-radtreelistview-to-your-application)

```

Comment:
Can you name those "Adding RadTreeListView to the Project" and "Populating with Data" to be consistent with the rest of the getting started articles we're updating?